sql/src/Main.sk

```diff
@@ -573,7 +573,7 @@ fun execDiff(args: Cli.ParseResults, options: SKDB.Options): void {
 
     for (dirSub in sub.dirSubs) {
       tableName = P.Name::create(
-        dirSub.dirName.dirName.stripPrefix("/").stripSuffix("/"),
+        dirSub.dirName.toString().stripPrefix("/").stripSuffix("/"),
```

Comment:
Do we expect `dirName.dirName` and `dirName.toString()` to be semantically equivalent?

Reply:
That's a good point.
As said offline:
- using `dirName.toString()` is already a bad pattern
- it's probably covered by tests
- and only in `sql/`
Let's not worry?